File: tugas keamanan/web/balok/balok.php
<?php
if(isset($_GET["Hitung"])){
	$p=$_GET['p'];
	$l=$_GET['l'];
	$t=$_GET['t'];
	$volume=$p*$l*$t;
	$luas=2*(($p*$l)+($p*$t)+($l*$t));
	$hasil;
	if ($p <0||$l <0||$t <0){
		$hasil= "Data tidak sesuai, mohon cek kembali<br>";
	} 
	else {
		$hasil= "{$volume}<br>${luas}";
	}
}
?>

<!DOCTYPE html>
<html>
<head>
	<title>Volume dan Luas Balok</title>
</head>
<body>
	<div align="center">
	<form action="" method="GET" >
	<h1>Menghitung Volume dan Luas Balok</h1><br>
	<p>Masukkan Panjang</p>
	<input type="number" name="p">
	<p>Masukkan Lebar</p>
	<input type="number" name="l">
	<p>Masukkan Tinggi</p>
	<input type="number" name="t"><br>
	<input type="Masukkan" name="Hitung">
	<br>
	<br>
	<h1 align="center">
		<?php 
			if (isset($_GET["Hitung"])) {
				echo $hasil;
			}
		?>
	</h1>
</form>
</div>
</body>
</html>

File: tugas keamanan/c++/balok.cpp
#include <iostream>
using namespace std;
 
 
float luas_permukaan(float p, float l, float t){
    float L;
    L = 2 * ( (p*l) + (p*t) + (l*t) );
    return L;
}
 
float volume(float p, float l, float t){
    float V;
    V = p * l * t;
    return V;
}
 
float keliling(float p, float l, float t){
    float k;
    k = 4 * (p + l + t);
    return k;
}
 
int main() {
    float p, l, t;
    float luasnya, volumenya, kelilingnya;
    int pilihan;
 
    cout << "Program menghitung luas, volume, dan keliling balok \n";
    cout << "Masukan pilihan. 1. luas, 2. volume, 3. keliling: ";
    cin >> pilihan;
 
    cout << "masukan panjang balok: ";
    cin >> p;
 
    cout << "masukan lebar balok: ";
    cin >> l;
 
    cout << "masukan tinggi balok: ";
    cin >> t;
 
    if (pilihan == 1) {
        luasnya = luas_permukaan(p, l, t);
        cout << "Jadi luasnya: " << luasnya;
    }
    else if (pilihan == 2) {
        volumenya = volume(p, l, t);
        cout << "Jadi volumenya: " << volumenya;
    }
    else if (pilihan == 3) {
        kelilingnya = keliling(p, l, t);
        cout << "jadi kelilingnya: " << kelilingnya;
    }
 
    return 0;
}
